docker-compose.local.yml

```diff
@@ -11,8 +11,8 @@ services:
       IROHA_CONFIG: /config/config.json
       IROHA_PUBLIC_KEY: ed01208BA62848CF767D72E7F7F4B9D2D7BA07FEE33760F79ABE5597A51520E292A0CB
       IROHA_PRIVATE_KEY: '{"digest_function":"ed25519","payload":"8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb"}'
-      TORII_P2P_ADDR: iroha0:1337
-      TORII_API_URL: iroha0:8080
+      TORII_P2P_ADDR: 0.0.0.0:1337
```

Comment:
why this change?

Reply:
It's required to bind local container address to localhost to enable the health check command. Otherwise container always receive a random local docker network address.